packages/manager/modules/backup-agent/src/pages/services/dashboard/agents/_components/__tests__/AgentIpsCell.components.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it, vi } from 'vitest';
+
+import {AgentIpsCell} from "@/pages/services/dashboard/agents/_components";
+
+vi.mock('@ovh-ux/manager-react-components', () => ({
+  DataGridTextCell: ({ children }: { children: React.ReactNode }) => (
+    <div data-testid="cell">{children}</div>
```

Comment:
Not sure the `data-testid` is necessary here since you don't use it in the test
```suggestion
    <div>{children}</div>
```